### PR TITLE
refac: MAR-344  refac: "< >" functionality of today and this week & ci package name updates

### LINKED ISSUE #700: refac: "< >" functionality of today and this week

### PR DESCRIPTION
**Pull request by immadsahin**
# What did you ship?

**Fixes:**

- [ ] #XXX (GitHub issue number)
- [ ] MAR-XXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

# Checklist:
- [ ] I have self-reviewed the code (A decent size PR without self-review might be rejected)
- [ ] I pinky swear that my codes gonna work as I have testing every possible scenario. 
- [ ] I ignored Coderabbit suggestion because it does not make any sense.
- [ ] I took Coderabbit suggestion under consideration as some of it makes sense.
- [ ] I have commented my code, particularly in hard-to-understand areas.

# OR:

- [ ] shut up and let me cook.

----

> [!IMPORTANT]
> Update project metadata, switch to bun for package management, and enhance UI interactions in `ThisWeekArrows.tsx` and `Date.tsx`.
> 
>   - **Metadata Updates**:
>     - Rename project from `satellite` to `march` in `package.json`.
>     - Update `description`, `author`, `repository`, and `bugs` fields in `package.json`.
>   - **Package Management**:
>     - Replace `npm` commands with `bun` commands in `README.md`.
>   - **UI Enhancements**:
>     - Modify `ThisWeekArrows.tsx` to improve button visibility and interaction for week navigation.
>     - Update `Date.tsx` to enhance date navigation with better button interactions.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=emptyarrayai%2Fmarch&utm_source=github&utm_medium=referral)<sup> for 5b09707b8b16db7127a182db04a15533aa815337. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

File: apps/frontend/src/components/ThisWeek/ThisWeekPage.tsx [hunk 1]
```diff
@@ -31,6 +31,8 @@ export const ThisWeekPage: React.FC = () => {
 
   const doneItems = items.filter((item: CycleItem) => item.status === "done")
 
+  const isCurrentWeek = weekNumber === getCurrentWeek(new Date())
+
   const handleWeekChange = (direction: "left" | "right" | "this") => {
```
[the remark this answers is not in the document]
Comment (immadsahin) [reply apps/frontend/src/components/ThisWeek/ThisWeekPage.tsx:37]: @ellipsis-dev fix

File: apps/frontend/src/components/ThisWeek/ThisWeekPage.tsx [hunk 2]
```diff
@@ -31,18 +31,22 @@ export const ThisWeekPage: React.FC = () => {
...
   const doneItems = items.filter((item: CycleItem) => item.status === "done")
 
+  const [isCurrentWeek, setIsCurrentWeek] = useState(weekNumber === getCurrentWeek(new Date()))
+
   const handleWeekChange = (direction: "left" | "right" | "this") => {
     setCurrentDate((prevDate) => {
       if (direction === "this") {
         const currentDate = new Date()
         const currentWeekNumber = getCurrentWeek(currentDate)
+        setIsCurrentWeek(currentWeekNumber === getCurrentWeek(new Date()))
```
[the remark this answers is not in the document]
Comment (oliursahin) [reply apps/frontend/src/components/ThisWeek/ThisWeekPage.tsx:41]: @ellipsis-dev fix this : Avoid calling getCurrentWeek(new Date()) multiple times. Store the result in a variable to improve performance and readability.

[the remark this answers is not in the document]
Comment (oliursahin) [reply apps/frontend/src/components/ThisWeek/ThisWeekPage.tsx:41]: > Avoid calling `getCurrentWeek(new Date())` multiple times. Store the result in a variable to improve performance and readability.

@ellipsis-dev fix this

[the remark this answers is not in the document]
Comment (oliursahin) [reply apps/frontend/src/components/ThisWeek/ThisWeekPage.tsx:41]: @elllipsis-dev can you address this comment : https://github.com/marchhq/march/pull/701#discussion_r1887940834

Comment (oliursahin) [inline apps/frontend/src/components/ThisWeek/ThisWeekPage.tsx:41]: @ellipsis-dev resolve it

**Review by deewakar-k — DISMISSED** [review]

File: apps/frontend/package.json [hunk 3]
```diff
@@ -114,10 +114,10 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/nirnejak/satellite-frontend"
+    "url": "https://github.com/marchhq/march"
```
Comment (sajdakabir) [inline apps/frontend/package.json:117]: please check the url

File: apps/backend/package.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
   "version": "0.0.1",
-  "description": "march Backend",
-  "author": "march-dev",
+  "description": "second brain for next generation makers",
+  "author": "emptyarray inc",
   "private": true,
   "main": "index.js",
   "type": "module",
-  "repository": "git@github.com:marchhq/march.git",
+  "repository": "https://github.com/marchhq/march.git",
```
Comment (sajdakabir) [inline apps/backend/package.json]: both URLs are different

**Review by sajdakabir — CHANGES_REQUESTED** [review]

**Review by sajdakabir — APPROVED** [review]

**Pull request merged**